desk/src/pages/desk/team/TeamSingle.vue

```diff
@@ -43,7 +43,9 @@
                 </template>
               </Button>
             </div>
-            <div v-else class="text-base text-gray-900">🙇 Such empty</div>
+            <div v-else class="text-base text-gray-900">
+              No Members found in Team: {{ teamId }}
```

Comment:
User lowercase - `No members found in team`

Reply:
@ssiyad Done, changed to lower case

Reply:
please check its done